src/main/java/teammates/ui/webapi/CreateAccountRequestAction.java

```diff
@@ -14,36 +15,30 @@
 class CreateAccountRequestAction extends AdminOnlyAction {
 
     @Override
-    public JsonResult execute() throws InvalidHttpRequestBodyException, InvalidOperationException {
+    public JsonResult execute()
+            throws InvalidHttpRequestBodyException, teammates.ui.webapi.InvalidOperationException {
```

Comment:
Let's not make this change of moving InvalidOperationException to logic layer for now. I can see that other actions in this PR are still throwing exceptions which should have been handled at the logic layer, so this change is more complicated than just moving `InvalidOperationException`

Reply:
InvalidOperationException has a special meaning within teammates. This is an error representing HTTP 409. Business logic shouldn't need to be aware of HTTP errors, that's action layer stuff, which is why this exception wasn't placed in the commons package.